app/viewmodels/SelfAssessmentPayment.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import org.joda.time.LocalDate
+
+case class SelfAssessmentPayment(date: LocalDate, referenceNumber: String, amount: Double) {
+
+  def getDisplayDate: String = date.toString("d MMMM")
```

Comment:
This wont work for Welsh. 
Look at https://github.com/hmrc/taxcalc-frontend/blob/master/app/utils/DisplayHelper.scala#L86 for example